lib/dal/src/diagram.rs

```diff
@@ -20,7 +20,6 @@ use crate::{
 };
 
 pub mod connection;
-pub mod node;
```

Comment:
The `node` module was useful when the diagram node abstraction was separate. Now that everything runs through `summary_diagram`, we co-locate all of that information in _private_ structs.